addon/utils/utils.js

```diff
@@ -74,7 +74,8 @@ export function getValidatableValue(value) {
   }
 
   if (isDSManyArray(value)) {
-    return emberArray(value.filter((v) => isValidatable(v)));
+    value.content = value.content.filter((v) => isValidatable(v));
```

Comment:
I'm not sure if this is an acceptable solution

Reply:
@runspired can you help?

Reply:
I found this on the qonto fork https://github.com/qonto/ember-cp-validations/commit/82a581661138645ebf7cbe7e6f1e7562ee1ff7c9

Reply:
folks need to resolve the promise array upfront, either by awaiting or by using `record.hasMany('foo').value()` to get it synchronously